docs/output.md

```diff
@@ -216,6 +222,23 @@ Output Summaries:
 
 [Bakta](https://github.com/oschwengers/bakta) is a tool for the rapid & standardised annotation of bacterial genomes and plasmids from both isolates and MAGs. It provides dbxref-rich, sORF-including and taxon-independent annotations in machine-readable JSON & bioinformatics standard file formats for automated downstream analysis. The output is used by some of the functional screening tools.
 
+### Functional classifications
```

Comment:
What about 'domainprediction' or something like that?

Reply:
We went for 'protein annotation' instead at the end.